Editor/EditorCore/PolyShapeEditor.cs

```diff
@@ -40,7 +40,7 @@ PolyShape polygon
             get { return target as PolyShape; }
         }
 
-        Material CreateHighlightLineMaterial()
+        static Material CreateHighlightLineMaterial()
```

Comment:
How this change affects this PR?

Reply:
It's not directly related, but it should be static and it seems wasteful to open a new pr for it :)

Reply:
Okey, just wanted to make that wasn't part of another PR :)